android/expoview/src/main/java/host/exp/exponent/network/ExponentNetwork.java

```diff
@@ -88,13 +96,29 @@ public OkHttpClient getNoCacheClient() {
   }
 
   public Cache getCache() {
-    int cacheSize = 40 * 1024 * 1024; // 40 MiB
-
-    // Use getFilesDir() because it gives us much more space than getCacheDir()
-    final File directory = new File(mContext.getFilesDir(), CACHE_DIR);
+    int cacheSize = 50 * 1024 * 1024; // 50 MiB
+    final File directory = new File(mContext.getCacheDir(), CACHE_DIR);
     return new Cache(directory, cacheSize);
   }
 
+  private void clearLegacyCache() {
+    if (mExponentSharedPreferences.getBoolean(ExponentSharedPreferences.HAS_CLEARED_LEGACY_OKHTTP_CACHE_KEY)) {
+      return;
+    }
```

Comment:
Instead of adding a key to every app (including those that never used the legacy cache), what if we just bailed out if the legacy cache directory doesn't exist?

Reply:
If we do this, then if any app tries to create a file or directory called `okhttp` in `FileSystem.documentDirectory`, it will always get deleted by this code. That is probably unlikely to happen -- but I think that adding this key to shared preferences is a little less destructive/unpredictable.